### PR TITLE
Pin internal dependencies

### PR DESCRIPTION
**Pull request by arnaudsjs**
# Description

Pin internal dependencies.

Part of inmanta/irt#528

### CODE REVIEW

**Review by sanderr — COMMENTED** [review]
I agree with what you stated on slack. I don't think this is required for the component repos.

**Review by sanderr — APPROVED** [review]
I was too quick to review this. This is not just a component dependency, this is a bit of an edge case. I guess it should be frozen here after all.

**Comment by arnaudsjs** [comment]
> I was too quick to review this. This is not just a component dependency, this is a bit of an edge case. I guess it should be frozen here after all.

Why?

**Comment by arnaudsjs** [comment]
Just for the scenario where an end-user install `inmanta-core` instead of the product package (`inmanta`) to obtain access to the cli tools?

File: tests_common/setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 requires = [
     "asyncpg",
     "click",
-    "inmanta-core~=4.0.dev",
+    "inmanta-core==4.1.0",
```
Comment (wouterdb) [inline tests_common/setup.py:25]: should we factor this out into a variable? 

```
version = "4.1.0"

requires = [
  f"inmanta-core=={version}",
]

setup(
    version=version,

**Review by wouterdb — COMMENTED** [review]
see my comment

File: tests_common/setup.py (hunk 1 above)
Comment (arnaudsjs) [reply tests_common/setup.py:25]: Sounds like a good idea.

**Review by wouterdb — APPROVED** [review]

**Pull request merged**